### PR TITLE
[11.0][OU-ADD] stock_landed_costs: Migration scripts

### PR DESCRIPTION
**Pull request by pedrobaeza**
@Tecnativa TT30777

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I think the error is due to the outdated .travis.yml definition:

https://app.travis-ci.com/github/OCA/OpenUpgrade/builds/241805488#L538

@MiquelRForgeFlow do you remember how to fix it?

**Comment by MiquelRForgeFlow** [comment]
Selection_051

**Comment by pedrobaeza** [comment]
I was about to tell you that I saw it. Pushed the fixed flake8 version.

File: addons/stock_landed_costs/migrations/11.0.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        env.cr, """
+        UPDATE stock_move sm
+        SET landed_cost_value = sub.landed_cost,
+            price_unit = price_unit + sub.landed_cost
+        FROM (
+            SELECT SUM(additional_landed_cost) AS landed_cost, move_id
+            FROM stock_valuation_adjustment_lines
+            GROUP BY move_id
+        ) sub
+        WHERE sub.move_id = sm.id AND sub.landed_cost != 0
```
Comment (MiquelRForgeFlow) [inline addons/stock_landed_costs/migrations/11.0.1.1/post-migration.py:20]:
```suggestion
        WHERE sub.move_id = sm.id AND sub.landed_cost != 0.0
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: addons/stock_landed_costs/migrations/11.0.1.1/post-migration.py (hunk 1 above)
Comment (pedrobaeza) [reply addons/stock_landed_costs/migrations/11.0.1.1/post-migration.py:20]: Why? It performs the same AFAIK.

Comment (MiquelRForgeFlow) [reply addons/stock_landed_costs/migrations/11.0.1.1/post-migration.py:20]: ok

**Pull request merged**